### ISSUE TITLE
Autocomplete works only on Post type forms

### ISSUE THREAD

**Issue by aposidelov** [CMB2/CMB2-Snippet-Library#86]
For example with 'object_types' => array( 'options-page' ) js error comes up: 

`$(...).autocomplete is not a function`

Probably the problem is that plugin's js script runs before jquery-ui-autocomplete.js library is added.

And on post type forms it works because autocomplete library is already there and it's used for adding  Tags.

**Comment by tw2113** [comment]
May just need to enqueue manually in this case.

**Comment by tw2113** [comment]
Looks like the script enqueuing in the sample file is using the wrong handle.

Wrong:
```
wp_enqueue_script( 'wp-jquery-ui-autocomplete' );
```
Correct:
```
wp_enqueue_script( 'jquery-ui-autocomplete' );
```

See: https://developer.wordpress.org/reference/functions/wp_enqueue_script/#default-scripts-included-and-registered-by-wordpress

**Comment by tw2113** [comment]
Fixed in 35a6d51
